content/docs-beta/SUSHI/migration/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Migrating from Older Versions"
+weight: 30
+---
+
+## Migrating from pre-1.0.0 Projects
+
+The SUSHI project structure and configuration changed significantly between pre-1.0.0 versions of SUSHI and versions of SUSHI 1.0.0 or greater. This guide will assume you currently have a project structure similar to the **customized ig** shown in the [project structure documentation](/docs/sushi/project/#ig-projects) for pre-1.0.0 versions of SUSHI. To migrate this project, the following changes are required:
+
+#### 1 - Move config.yaml
+Move the **customized-ig/fsh/config.yaml** file to the top-level of the **customized-ig** folder, and rename the file to **sushi-config.yaml**, so that the file is located at **customized-ig/sushi-config.yaml**.
+#### 2 - Move .fsh files
+Move all **.fsh** files within the **customized-ig/fsh** folder to a new folder, **customized-ig/input/fsh**.
+#### 3 - Move ig-data Content
+Move the contents of the **customized-ig/fsh/ig-data** folder to their corresponding location within the **customized-ig** folder. When moving these files, each file should be placed at a location under **customized-ig** that corresponds to the file's previous location under **ig-data**. So for example, if you had an **ig.ini** file at **customized-ig/fsh/ig-data/ig.ini**, that file should be moved to just **customized-ig/ig.ini**. If you had a **menu.xml** file at **customized-ig/fsh/ig-data/includes/menu.xml**, that file should be moved to **customized-ig/includes/menu.xml**.
+#### 4 - Remove fsh/ Directory
+At this point, all of the contents of the **customized-ig/fsh** folder should have been moved out of **customized-ig/fsh**, so that folder can be safely deleted.
+#### 5 - Update Content of sushi-config.yaml
+Update **sushi-config.yaml** to remove the `template` property if necessary. The `template` property is used by SUSHI to generate an **ig.ini** file, and its usage is now deprecated. The **ig.ini** file must be managed directly. If SUSHI detects usage of the `template` property to generate an **ig.ini** file, it will log an error message containing the **ig.ini** that would be generated by that `template` property. Copy these contents, and place them in an **ig.ini** file located at **customized-ig/ig.ini**.
+
+{{% alert title="Warning" color="warning" %}}
+The **ig.ini** file MUST use the `#current` version of the `fhir.base.template` (or a template that extends this version). So to use the `fhir.base.template`, the `template` property in **ig.ini** will be `template = fhir.base.template#current`.
```

Comment:
I think we should mention that this is a temporary limitation and we should be more explicit about templates allowed. How about this?

* As of the SUSHI 1.0 Beta release, the **ig.ini** file MUST use a template based on `fhir.base.template#current`. Specific template versions (i.e., other than `#current`) are expected to work in the future.  For now, any of the following should work:
  * `template = fhir.base.template#current`
  * `template = hl7.base.template#current`
  * `template = hl7.fhir.template#current`
  * `template = hl7.davinci.template#current`
  * `template = hl7.cda.template#current`